src/pudl/transform/eiaaeo.py

```diff
@@ -514,6 +514,87 @@ def core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type(
     return renamed_for_pudl
 
 
+@asset(io_manager_key="pudl_io_manager")
+def core_eiaaeo__yearly_projected_delivered_energy(
+    raw_eiaaeo__energy_consumption_by_sector_and_source,
+):
+    """Projected delivered energy for commercial, industrial, residential, and transportation sectors.
+
+    The "Delivered Energy" series in Table 2 are a subtotal, and include purchased
+    electricity, renewable energy, and an array of fuels based on sector. It
+    explicitly excludes electricity-related losses. There is detailed information in
+    the footnotes of the online data browser:
+
+      https://www.eia.gov/outlooks/aeo/data/browser/#/?id=2-AEO2023
+
+    In that link, subtotal series are displayed indented, and include all lines above
+    them which are one level out, up to the next indented line. Delivered Energy is a
+    special case which includes those plus the subtotals above it.
+
+    AEO Delivered Energy figures are variously referred to as delivered energy,
+    energy consumption, energy use, and energy demand, depending on which org and
+    which document is describing them. In PUDL we say delivered energy or energy
+    consumption.
+    """
```

Comment:
hmm maybe the above belongs in the table description metadata instead? or both here and in the description?